kornia/augmentation/_2d/geometric/perspective.py

```diff
@@ -23,7 +23,7 @@ class RandomPerspective(GeometricAugmentationBase2D):
         align_corners: interpolation flag.
         keepdim: whether to keep the output shape the same as input (True) or broadcast it
                  to the batch form (False).
-
+        area_preserving: if ``True``, applies random offsets which preserve image area on average.
```

Comment:
A blank line is required
```suggestion
        area_preserving: if to apply random offsets which preserve image area on average.

```